backend/config/autoload/mail.global.dev.php

```diff
@@ -3,9 +3,10 @@
 return [
     'laminas_mail' => [
         'transport' => [
-            'type' => 'file',
+            'type' => 'smtp',
```

Comment:
@pmattmann I didn't find out what this option does/whether it is necessary

Reply:
It is used to configure what type of mail transfer to use and where the SMTP server lies. See my attempt of the same here: https://github.com/carlobeltrame/ecamp3/commit/9491608be8aa24537d81a653f502b9a255304283

Reply:
Ok, I wasn't sure what to put there as `type` but `smtp` seems to work